src/style.css

```diff
@@ -1,3 +1,86 @@
+:root {
+  --color-hover: #00acdc;
+}
+
 body {
   margin: 0;
 }
+
+html {
+  font-family: Roboto, Arial, Helvetica, sans-serif;
+  font-size: 12px;
+  font-style: normal;
+  font-weight: 500;
+}
+
+.img-class {
+  overflow-clip-margin: content-box;
+  overflow: clip;
+}
+
+.img {
+  width: 40px;
+  height: 40px;
+  display: flex;
+}
```

Comment:
The `.img` class is defined with `display: flex;` but it doesn't seem to have any child elements to align. If the class is used for an `img` tag, `display: flex;` won't have any effect because images are replaced elements. If there are no child elements to align, you can remove this property.